stage4.py

```diff
@@ -0,0 +1,48 @@
+from card import Card
+import random
+
+
+deck = list()
+cards_in_deck = 0
+is_card_identical = 0
+while cards_in_deck < 40:
+    pulled_card = Card()
+    for card in deck:
+        if card == pulled_card:
+            is_card_identical = 1
+            break
+    if is_card_identical == 1:
+        is_card_identical = 0
+        continue
+    else:
+        cards_in_deck += 1
+        deck.append(pulled_card)
+
+random.shuffle(deck)
+
+num_players = 0
+while int(num_players) not in range(2, 7):
```

Comment:
Why did you convert the variable to int? (I know it is a string) 
the name num_players sounds like it should be an int. 
change the name of the variable or the code.